sources/packages/backend/apps/api/src/route-controllers/assessment/assessment.controller.service.ts

```diff
@@ -104,7 +104,14 @@ export class AssessmentControllerService {
     }
 
     return {
-      assessment: assessmentDTO,
+      assessment: {
+        ...assessmentDTO,
+        totalAdditionalTransportationAllowance:
+          assessment.workflowData?.calculatedData
+            ?.totalAdditionalTransportationAllowance,
+        returnTransportationCost:
+          assessment.workflowData?.calculatedData?.returnTransportationCost,
```

Comment:
Do we require null check? workflow data must always be there when an assessment is completed. Let me know if I am missing something.